<<<<<<< HEAD

syntax = "proto3";

message PublicAggreDealsV3ApiItem {
  string price = 1;
  string quantity = 2;
  int32 tradeType = 3;
  int64 time = 4;
}

message PublicAggreDealsV3Api {
  repeated PublicAggreDealsV3ApiItem deals = 1;
  string eventType = 2;
}

message PushDataV3ApiWrapper {
  string channel = 1;
  PublicAggreDealsV3Api publicAggreDeals = 314;
  string symbol = 3;
}
=======

syntax = "proto3";

message PublicAggreDealsV3ApiItem {
  string price = 1;
  string quantity = 2;
  int32 tradeType = 3;
  int64 time = 4;
}

message PublicAggreDealsV3Api {
  repeated PublicAggreDealsV3ApiItem deals = 1;
  string eventType = 2;
}

message PushDataV3ApiWrapper {
  string channel = 1;
  PublicAggreDealsV3Api publicAggreDeals = 314;
  string symbol = 3;
}
>>>>>>> 56955970207bc584405376f062e416679b035c00
    